File: hyeonsook95/baekjoon/9095.py
import sys

input = sys.stdin.readline


def solution(N):
    def dfs(number, count):
        if number == 0:
            return count + 1

        for num in [1, 2, 3]:
            if number - num >= 0:
                count = dfs(number - num, count)
        return count

    return dfs(N, 0)


if __name__ == "__main__":
    T = int(input())
    for _ in range(T):
        print(solution(int(input())))

File: hyeonsook95/baekjoon/9095.ts
var fs = require("fs");
var input = fs.readFileSync('dev/stdin').toString().split('\n');
const T = parseInt(input.shift());

function dfs(number, count) {
    if (number == 0) {
        return count + 1
    }

    for (var num of [1, 2, 3]) {
        if (number >= num) {
            count = dfs((number - num), count);
        }
    }
    return count;
}

for (var t = 0; t < T; t++) {
    var number = parseInt(input[t]);
    console.log(dfs(number, 0));
}